services/ext/service.go

```diff
@@ -442,6 +444,7 @@ func buildMessengerOptions(
 		protocol.WithWalletConfig(&config.WalletConfig),
 		protocol.WithWalletService(walletService),
 		protocol.WithCommunityTokensService(communityTokensService),
+		protocol.WithWakuService(wakuService),
```

Comment:
Probably non an issue since we only use WakuV2, but i think this can be nil if using WakuV1